Upcoming-iCal-Events.widget/main.coffee

```diff
@@ -96,33 +127,27 @@ update: (output, domEl) ->
             nameAndCalendar = lines[i].split('(')
 
             if nameAndCalendar.length < 2
-            	continue
+                continue
 
             name = nameAndCalendar[0].replace(bullet, '')
             calendar = 'No Calendar'
 
             if nameAndCalendar[1] != undefined
                 calendar = nameAndCalendar[1].replace(')','')
 
-            if IGNORE_CALENDER.includes(calendar)
-                continue
-
             if ( name.length > MAX_CHARACTERS )
                 name = name.substr(0, MAX_CHARACTERS) + "..."
 
-            date = ((lines[i+1].split("at"))[1])
-            if ( date == undefined )
-                date = "ALLDAY"
-            else
-                date = date.substr(0,9)
+            datePrefix = if (day && not USE_LATER_DATE) then day + ' ' else ''
+
+            date = ((lines[i+1].split("at"))[1]) or ALL_DAY_EVENT_LABEL
```

Comment:
how does this handle events with "at" in the title? Seeing some issues with then incorrectly labelled as all-day events.

Reply:
Since we are fetching only the Event name and the date and time, lines[i+1] should have only the date and time.